script.js

```diff
@@ -11,4 +11,29 @@ function toggleTheme() {
   const ielement = document.getElementById("myInput");
   ielement.classList.toggle("Dark")
 }
-function newele() { }
+
+function newElement() {
+  var li = document.createElement("li");
+  var inputValue = document.getElementById("myInput").value;
+  var t = document.createTextNode(inputValue);
+  li.appendChild(t);
+  if (inputValue =='') {
+    alert("please fill the text box with letters ");
+  } else {
+    document.getElementById("Todo").appendChild(li);
+  }
+  document.getElementById("myInput").value = "";
+
+  var span = document.createElement("SPAN");
+  var txt = document.createTextNode("\u00D7");
+  span.className = "close";
+  span.appendChild(txt);
+  li.appendChild(span);
+
+  for (i = 0; i < close.length; i++) {
+    close[i].onclick = function() {
+      var div = this.parentElement;
+      div.style.display = "none";
+    }
+  }
+}
```

Comment:
read about es6 features, then refactor this function to make use of const, let and arrow functions